jdhzzz-CT-USA.md

```diff
@@ -0,0 +1,11 @@
+# Central CT
```

Comment:
Can you add example hometown name, please? I'm overseas in the UK and have no idea what it means Central CT